src/components/LoansByCategory/MFIRecommendations/MFIHero.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+	<div class="tw-w-full">
+		<div class="tw-flex tw-items-center tw-gap-1 tw-px-2 lg:tw-px-0">
+			<img class="tw-w-4 lg:tw-w-6" src="@/assets/images/mfi_recommendations/earth.svg" alt="">
+			<h3 class="tw-text-h3 tw-text-primary tw-whitespace-nowrap">
+				Field Partner Spotlight
+			</h3>
+			<hr class="tw-border-tertiary tw-w-full tw-ml-2 tw-hidden lg:tw-block">
+		</div>
+		<div class="tw-mt-2 lg:tw-mt-4 tw-px-2">
+			<h2 class="tw-text-h2 tw-text-primary">
+				Fundación Pro Mujer
```

Comment:
It looks like this addition to the page just focused on a single static partner and is not behind an experiment. Is that expected? I would have assumed we'd want the data to be more dynamic if it wasn't an experiment.

Reply:
Thanks @dyersituations !
Experiment condition was left, fixed now!